matchms/filtering/make_charge_scalar.py

```diff
@@ -12,4 +12,8 @@ def make_charge_scalar(spectrum_in: SpectrumType) -> SpectrumType:
     if isinstance(spectrum.get("charge", None), list):
         spectrum.set("charge", int(spectrum.get("charge")[0]))
 
+    # convert string charges to int
+    elif isinstance(spectrum.get("charge", None), str):
```

Comment:
```suggestion
    if isinstance(spectrum.get("charge", None), str):
```

Reply:
No need for `elif` here I think. With `if` it would also cover something that was first a list and then still a string (after the step above your code) .

Reply:
good point